test/e2e/set/scyllacluster/multi_datacenter_scyllacluster_external_seeds.go

```diff
@@ -0,0 +1,327 @@
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+		}
+
+		framework.By("Ensuring default namespace presence in the first cluster")
+		ns1, ns1Client, ok := f.Cluster(0).DefaultNamespaceIfAny()
+		o.Expect(ok).To(o.BeTrue())
+
+		framework.By("Creating first ScyllaCluster")
+		sc1, err := ns1Client.ScyllaClient().ScyllaV1().ScyllaClusters(ns1.GetName()).Create(ctx, sc1, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the first ScyllaCluster to rollout (RV=%s)", sc1.ResourceVersion)
+		waitCtx1, waitCtx1Cancel := utils.ContextForRollout(ctx, sc1)
+		defer waitCtx1Cancel()
+		sc1, err = controllerhelpers.WaitForScyllaClusterState(waitCtx1, ns1Client.ScyllaClient().ScyllaV1().ScyllaClusters(sc1.Namespace), sc1.Name, controllerhelpers.WaitForStateOptions{}, utils.IsScyllaClusterRolledOut)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		verifyScyllaCluster(ctx, ns1Client.KubeClient(), sc1)
+		waitForFullQuorum(ctx, ns1Client.KubeClient().CoreV1(), sc1)
+
+		hosts1, hostIDs1, err := utils.GetBroadcastRPCAddressesAndUUIDs(ctx, ns1Client.KubeClient().CoreV1(), sc1)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		di1 := insertAndVerifyCQLData(ctx, hosts1)
+		defer di1.Close()
+
+		sc2 := f.GetDefaultScyllaCluster()
+		sc2.Name = "multi-datacenter-cluster"
+		sc2.Spec.Datacenter.Name = "dc2"
+		sc2.Spec.Datacenter.Racks = []scyllav1.RackSpec{
+			{
+				Name:    "a",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+			{
+				Name:    "b",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+			{
+				Name:    "c",
+				Members: 1,
+				Storage: scyllav1.Storage{
+					Capacity: "1Gi",
+				},
+				Resources: corev1.ResourceRequirements{
+					Limits: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("1"),
+						corev1.ResourceMemory: resource.MustParse("1Gi"),
+					},
+					Requests: corev1.ResourceList{
+						corev1.ResourceCPU:    resource.MustParse("10m"),
+						corev1.ResourceMemory: resource.MustParse("100Mi"),
+					},
+				},
+			},
+		}
+		sc2.Spec.ExternalSeeds = append(make([]string, 0, len(hosts1)), hosts1...)
```

Comment:
Does it really matter though? It's only used for discovery really (+when no known hosts are reachable)

Reply:
It doesn't matter for current setup where same network is used by both nodes and client. In future this can change and will break this. 
Using rpc_address is just wrong thing to do, so lets fix it.

Reply:
+1, it can be a different network range

Reply:
I'll have to send a fix for the existing external_seeds test first (it's copied over), I'll link it here

Reply:
https://github.com/scylladb/scylla-operator/pull/1984
and done here as well